TemporaryOysters/bangazon-cli
# update main menu navigation

##Pull Requests



##Descriptions:

* Please describe what issue this PR solves

updated main menu navigation for users on command line









##Steps to Test:
run "python CLI.py" and the menu should automatically show



##Related Issue:



##System configuration.


* Are there 3rd party libraries that need to be installed?


* Any command line utilities to run?
